### ISSUE TITLE
SA1001 and SA1113 reported in parameter lists

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1971]
The following code results in both SA1001 and SA1113.

```csharp
MethodName(
  x
  , y);
```

This was caused by #1663, and poses a substantial problem with the usability of SA1001 in cases where SA1113 is intentionally disabled.

The implementation is **up for grabs**, but I'm also adding **needs discussion** to determine if this change gets applied to 1.0.0 or 1.1.0.

**Comment by vweijsters** [comment]
:question: What exactly is the problem here? 

The only objection that one could have in my opinion is the double warning that gets reported, but both warnings are correct. Even disabling SA1113 should still leave the SA1001 one.

StyleCop classic reports SA1001 and SA1113 here as well. 

In my opinion this is behavior by design and nothing should be changed.

**Comment by sharwell** [comment]
We have a code base that used this style previously. We aren't ready yet to fix SA1113 yet, but we wanted to enable SA1001 to check all other uses of commas. This is the way we used beta006, but it changed since then and it's looking like we'll have to just disable the rule.

I'll check again today and see if the new baseline functionality in Update 1 will hold us over.

**Comment by sharwell** [comment]
Since the behavior matches StyleCop Classic, this can wait for the 1.1.0.

**Comment by vweijsters** [comment]
I'd prefer that you mark this as `needs discussion`, not as a `bug` and `up for grabs`, as this is a clear change of behavior that will need discussion. For me the example above is clearly a departure from what StyleCop stands for and it can therefore not be classified as bug.

**Comment by sharwell** [comment]
@vweijsters I rewrote the original post as a proposal.

My vote is :+1:

**Comment by vweijsters** [comment]
Thanks!

My vote will be :-1:
